synchronizer/synchronizer.go

```diff
@@ -272,11 +274,11 @@ func (d *Synchronizer) SyncCycle(customerDbUriFlag string) (bool, error) {
 			if latestErr != nil {
 				return isEnd, fmt.Errorf("failed to get latest block number: %v", latestErr)
 			}
-			d.startBlock = uint64(crawler.SetDefaultStartBlock(0, latestBlockNumber))
+			d.startBlock = uint64(crawler.SetDefaultStartBlock(0, latestBlockNumber)) - 100
```

Comment:
I would avoid hardcode shift each time I start, there are cases when you want to start from exact block